modin/config/envvars.py

```diff
@@ -219,6 +219,7 @@ class TestDatasetSize(EnvironmentVariable, type=str):
 
     varname = "MODIN_TEST_DATASET_SIZE"
     choices = ("Small", "Normal", "Big")
+    default = "Small"
```

Comment:
dataset size for modin tests also depends on that variable, before this changes tests were using `"Normal"` size by default, now it will be `"Small"`
https://github.com/modin-project/modin/blob/9cb165dd889b9507820c84532b5ba613ac5edb9a/modin/pandas/test/utils.py#L42-L43